src/spec-node/featuresCLI/testCommandImpl.ts

```diff
@@ -300,9 +301,16 @@ async function generateDefaultProjectFromFeatures(
 	const tmpFolder = await createTempDevcontainerFolder(cliHost);
 
 	const features = featuresToTest
-		.map((x) => `"${collectionsDirectory}/src/${x}": {}`)
+		.map((x) => `"./${x}": {}`)
 		.join(',\n');
 
+	for (const featureId of featuresToTest) {
+		// Copy the feature source code to the temp folder
+		const pathToFeatureSource = `${collectionsDirectory}/src/${featureId}`;
+		await cpDirectoryLocal(pathToFeatureSource, `${tmpFolder}/.devcontainer/${featureId}`);
+		process.stdout.write('AFTER\n');
```

Comment:
Debugging output that should be removed before merging?

Reply:
Ah, thought i'd removed this. Thanks